src/server/Story2sketch.js

```diff
@@ -289,9 +327,12 @@ export default class Story2sketch {
       }
     });
 
-    this.positionSymbols();
-
-    fs.writeFileSync(this.output, JSON.stringify(this.sketchPage));
+    if (this.byKind) {
+      this.writeByKind()
+    } else {
+      this.positionSymbols();
```

Comment:
We should add a feature flag to toggle between `positionSymbolsByViewport()` and `positionSymbolsByKind()` - `layoutByKind`?

Reply:
layoutByKind added ✅